infrastructure/firecracker/config.go

```diff
@@ -217,9 +217,10 @@ func generateNetworkConfig(vm *models.MicroVM) (string, error) {
 		macAddress := getMacAddress(&iface, status)
 
 		eth := &cinetwork.Ethernet{
-			Match: cinetwork.Match{},
-			DHCP4: firecracker.Bool(true),
-			DHCP6: firecracker.Bool(true),
+			Match:          cinetwork.Match{},
+			DHCP4:          firecracker.Bool(true),
+			DHCP6:          firecracker.Bool(true),
+			DHCPIdentifier: firecracker.String("mac"),
```

Comment:
would be good to have that as a const, but otherwise LGTM

Reply:
Good idea, changed.